java/vector/src/main/java/org/apache/arrow/vector/types/IntervalUnit.java

```diff
@@ -17,8 +17,13 @@
 
 package org.apache.arrow.vector.types;
 
+/**
+ * Resolutions that intervals (periods can be stored in).
```

Comment:
something looks off in the comment..

Reply:
Reworded to:  Resolutions for Interval Vectors.